AppCenter-Windows.sln

```diff
@@ -970,6 +981,34 @@ Global
 		{2FFFF8C2-E190-4B83-9AD8-19F0E2AC6088}.Release|x64.Build.0 = Release|Any CPU
 		{2FFFF8C2-E190-4B83-9AD8-19F0E2AC6088}.Release|x86.ActiveCfg = Release|Any CPU
 		{2FFFF8C2-E190-4B83-9AD8-19F0E2AC6088}.Release|x86.Build.0 = Release|Any CPU
+		{A42BE92C-616D-4333-9F6A-D05BA547BFC6}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{A42BE92C-616D-4333-9F6A-D05BA547BFC6}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{A42BE92C-616D-4333-9F6A-D05BA547BFC6}.Debug|ARM.ActiveCfg = Debug|Any CPU
+		{A42BE92C-616D-4333-9F6A-D05BA547BFC6}.Debug|ARM.Build.0 = Debug|Any CPU
+		{A42BE92C-616D-4333-9F6A-D05BA547BFC6}.Debug|ARM64.ActiveCfg = Debug|Any CPU
+		{A42BE92C-616D-4333-9F6A-D05BA547BFC6}.Debug|ARM64.Build.0 = Debug|Any CPU
+		{A42BE92C-616D-4333-9F6A-D05BA547BFC6}.Debug|iPhone.ActiveCfg = Debug|Any CPU
```

Comment:
Not sure if we actually need those iPhone targets here

Reply:
No need for it, please remove

Reply:
Its VS doing that automatically, happened again on Mac solution when I reused the code for Xamarin forms. I guess it's ok...